Description: Patch for recent versions of glibc which always assume int64_t support.
Author: John Maddock
Origin: https://svn.boost.org/trac/boost/changeset/84950
Bug: https://svn.boost.org/trac/boost/ticket/8731
Last-Update: 2014-02-28

Index: /trunk/boost/cstdint.hpp
===================================================================
--- a/boost/cstdint.hpp	(revision 84805)
+++ b/boost/cstdint.hpp	(revision 84950)
@@ -42,5 +42,8 @@
 // See https://svn.boost.org/trac/boost/ticket/3548 and http://sources.redhat.com/bugzilla/show_bug.cgi?id=10990
 //
-#if defined(BOOST_HAS_STDINT_H) && (!defined(__GLIBC__) || defined(__GLIBC_HAVE_LONG_LONG))
+#if defined(BOOST_HAS_STDINT_H)					\
+  && (!defined(__GLIBC__)					\
+      || defined(__GLIBC_HAVE_LONG_LONG)			\
+      || (defined(__GLIBC__) && ((__GLIBC__ > 2) || ((__GLIBC__ == 2) && (__GLIBC_MINOR__ >= 17)))))
 
 // The following #include is an implementation artifact; not part of interface.
